integration-tests/src/test/java/oracle/kubernetes/operator/utils/Operator.java

```diff
@@ -312,6 +326,7 @@ public void callHelmInstall() throws Exception {
         .append(" --set \"imagePullPolicy=")
         .append(imagePullPolicy)
         .append("\" --wait --timeout 60");
+    TestUtils.exec("cat " + generatedInputYamlFile);
```

Comment:
If this only for debugging, can u use logger?

Reply:
removed it.